templates/main.template

```diff
@@ -100,165 +126,239 @@ Parameters:
       can include numbers, lowercase letters, uppercase letters, and hyphens (-).
-      It cannot start or end with a hyphen (-).
+      It cannot start or end with a hyphen (-). If you are unsure, do not change this value.
     Type: String
   QSS3KeyPrefix:
     AllowedPattern: ^[0-9a-zA-Z-/]*$
     ConstraintDescription: Quick Start key prefix can include numbers, lowercase letters,
-      uppercase letters, hyphens (-), and forward slash (/).
+      uppercase letters, hyphens (-), and forward slash (/). If you are unsure, do not change this value.
     Default: quickstart-compliance-pci/
     Description: S3 key prefix for the Quick Start assets. Quick Start key prefix
       can include numbers, lowercase letters, uppercase letters, hyphens (-), and
-      forward slash (/).
+      forward slash (/). If you are unsure, do not change this value.
     Type: String
 Mappings:
-  CustomVariables:
-    vResourceEnvironmentTagKey:
-      Value: Environment
-    vResourceEnvironmentTagValue:
-      Value: development
   RegionServiceSupport:
     ap-northeast-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-northeast-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-south-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-southeast-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-southeast-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ca-central-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     eu-central-1:
       NatGateway: 'true'
-      ConfigRules: 'true'
       Glacier: 'true'
     eu-west-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     eu-west-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     sa-east-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'false'
     us-east-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-east-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-gov-west-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-west-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-west-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
+#these have been updated to a 2019 version
   AWSAMIRegionMap:
     AMI:
-      AMZNLINUXHVM: amzn-ami-hvm-2018.03.0.20181129-x86_64-gp2
+      AMZNLINUXHVM: amzn-ami-hvm-2018.03.0.20190514-x86_64-gp2
     ap-northeast-1:
-      AMZNLINUXHVM: ami-00a5245b4816c38e6
+      AMZNLINUXHVM: ami-0ccdbc8c1cb7957be
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     ap-northeast-2:
-      AMZNLINUXHVM: ami-00dc207f8ba6dc919
+      AMZNLINUXHVM: ami-082b5ca9ff663f3b8
       InstanceType: m4.large
```

Comment:
Are you willing to move the `InstanceType` mappings to an alternate mapping? 
```
Mappings
  AWSAMIRegionMap:
    (ami_info)
  RegionInstanceTypeMap:
    (instance_type_info)
```

Reply:
I think we can actually remove the instance type, its all m4.large...